packages/headless/src/modules/auth/graphql/auth.model.ts

```diff
@@ -61,9 +61,6 @@ export class AdminSignUpResponse {
 
 @ObjectType()
 export class AdminSignInResponse {
-  @Field({ nullable: true })
-  error?: string;
-
```

Comment:
can't there be any error?

Reply:
No..Because we handle it in Graphql Init Function & also error handle in grapqhqlRespose in helper module.